controllers/front/PaymentCard16.php

```diff
@@ -68,7 +68,12 @@ public function initContent()
             'clientToken' => $paypalOrder['body']['client_token'],
             'paypalOrderId' => $paypalOrder['body']['id'],
             'validateOrderLinkByCard' => $module->getValidateOrderLink($paypalOrder['body']['id'], 'card'),
-            'intent' => strtolower(\Configuration::get('PS_CHECKOUT_INTENT')),
+            'intent' => strtolower(Configuration::get(
```

Comment:
we don't have strtolower there https://github.com/PrestaShopCorp/ps_checkout/pull/281/files#diff-3d415b068dfb8918e1bdee4e51078c7eR127
if we can avoid it everywhere it would be better IMO

Reply:
Done